src/libraries/Microsoft.PowerFx.Core/Public/Values/ErrorValue.cs

```diff
@@ -89,18 +96,39 @@ public override void ToExpression(StringBuilder sb, FormulaValueSerializerSettin
                 flag = false;
 
                 sb.Append("{Kind:ErrorKind.");
-                sb.Append(errorValue.Kind.ToString());
+                sb.Append(error.Kind.ToString());
 
-                if (errorValue.Message != null)
+                if (error.Message != null)
                 {
                     sb.Append(", Message:");
-                    sb.Append(CharacterUtils.ToPlainText(errorValue.Message));
+                    sb.Append(CharacterUtils.ToPlainText(error.Message));
                 }
 
                 sb.Append("}");
             }
 
             sb.Append("))");
         }
+
+        private void AppendCompactRepresentation(StringBuilder sb)
+        {
+            var flag = true;
+
+            sb.Append("Error(Kind=");
```

Comment:
>Error(Kind= [](http://example.com/codeflow?start=23&length=11)

Is the compact representation supposed to be easier to round trip? If so, we should use something like `Error({Kind:ErrorKind.<thekind>})` for a single error, or `Error(Table({Kind:ErrorKind.<kind1>},{Kind:ErrorKind.<kind2>},...))` for multiple errors.

Reply:
Error({Kind:ErrorKind.Div0}) is valid